README.md

```diff
@@ -51,6 +51,33 @@ Please refer to the installation section in [quick_start.md](docs/source/quick_s
 
 ## Get Started
 
+To immediately use a model on a given input image, we provide the Predictor API. Predictor group together a pretrained model with the preprocessing that was used during that model's training. For example, we can easily extract detected objects in an image:
```

Comment:
add this paragrath to detection tutorial

Reply:
done